Add KB: Troubleshooting: NFS server setting and mount a NFS storage (#503)
longhorn/website: content/kb/troubleshooting-nfs-server-setting-and-mount-nfs.md

Then the user tries to use the absolute path of the directory on NFS server to list longhorn backups, it is the cause of failure for NFSv4.

### Solution

1. Mount with the absolute path "/opt/nfs-server" in client side **without** the option "fsid=0" in server side.

2. Mount with the path "/" in client side and remain the option "fsid=0" in server side.

## Symptoms-2

The user setups the backup target on Setting Page and starts to backup volume and hits:

```text
error running create backup command: failed to create backup to nfs://192.168.121.170:/opt/nfs-server for volume test-for-backup: rpc error: code = Unknown desc = mkdir /var/lib/backupstore/192_168_121_170/opt/nfs-server/backup/longhorn/backupstore: permission denied" , error exit status 1
```

### Reason

The NFS client has no permission to access remote directory the NFS server shared. The directory is not accessiable by normal group/user and the option "root_squash" might be used for shared the directory.
### Solution

1. Use the option "no_root_squash" instead of "root_squash"

> 2. Add chmod o+w [shared directory path] or change the directory owner to `nobody`.

Reviewer: ```suggestion
2. Execute `chmod o+w [exported directory path]` or change the owner of the directory to `nobody`.
```

## Related information

> * Related Longhorn issue:</br>

Reviewer: ```suggestion
* Related Longhorn issues:
```

>   <https://github.com/longhorn/longhorn/issues/3576>
>   <https://github.com/longhorn/longhorn/discussions/3805>

Reviewer: ```suggestion

  - <https://github.com/longhorn/longhorn/issues/3576>
  - <https://github.com/longhorn/longhorn/discussions/3805>
```
